bin/move_files.py

```diff
@@ -175,4 +178,7 @@
     db.update_file(attr_name='name',
                    value=tup[1],
                    name=tup[0],
-                   dry=str2bool(args.dry))
\ No newline at end of file
+                   dry=str2bool(args.dry))
+    
+db.add_ticket_track(args.ticket, args.tg_dir)
```

Comment:
```suggestion
db.add_ticket_track(args.ticket, args.tg_dir, dry=str2bool(args.dry))
```

Reply:
`tg_dir` is not always defined by the user, when user runs the command with dest instead, `args.tg_dir` goes as NULL. We should either make`--tg_dir` mandatory or compute the directory from `dest`